capabilities/empowering-teams-to-choose-tools.md

```diff
@@ -1,51 +1,82 @@
+This capability is about empowering teams to select the tools and technologies that best support their unique workflows and tasks. The DORA research has found that the most qualified decision-makers are the individual contributors themselves. When teams are empowered to choose their own tools, we tend to see higher software delivery performance and increases in job satisfaction.
 
-## Nuance
+## Nuances
 
-### Balancing Freedom with Constraints
+### Balance Choice with Complexity
 
-It’s crucial to balance tool choice with the potential costs of acquisition, support, and added complexity in communication between teams using different tools. A baseline set of approved tools should be established, which can cover the majority of the organization's needs while allowing room for innovation. Typically, organizations have platform teams that do a lot of the "behind the scenes" support work required to keep a technology stack compliant and efficient.
+It’s crucial to balance tool choice with the potential costs of acquisition, support, and added complexity that come with adding new tools to a tech stack. 
 
-But if a team feels strongly that a different tool or technology is best for their case, they are free to choose it. They should know that their choice means they must also do the work of supporting this new tool or set of tools.
+While a baseline set of tools should be established across an organization, teams should feel free to choose a new tool or technology if they feel strongly that it is best suited for their use case. Making this choice means they must also _support_ the new tool(s), which can be a tall order.
 
-Sometimes that support work can be easy to gloss over. Take for example the decision to use a different language. The teams will need to build out a whole new CI/CD pipeline and ensure they have a environments suitable to run the code in the new language. They'll also need to introduce tooling that scans for security vulnerabilities and a host of other tasks that a company's platform team likely already handles.
+Take, for example, a team's choice to use a different language. Once this decision is made, the team will need to build a new CI/CD pipeline and ensure its environments are suitable to run their code in the new language. They'll also need to introduce other new tooling that scans for security vulnerabilities and a host of other tasks, which the organization's platform team likely already handles. After all, the role of the platform team is to keep the technology stack compliant and efficient. 
 
-### Risk of Tool Overproliferation
+As you can see, one team's decision to use another tool has created a domino effect in terms of support, and has potentially added a layer of complexity in communication between teams. 
 
-**Allowing teams to choose their tools doesn't mean unrestricted freedom.**
+### Develop Standards for Adopting New Tools
```

Comment:
This kinda feels like a practice. Thoughts?

Reply:
Ah yes. It seems we're having this conversation in two threads here. That works for me.